src/components/MaterialUI/Inputs/DatePicker/TimePicker.js

```diff
@@ -0,0 +1,169 @@
+import React, { useState, useCallback } from "react";
+import styled from "styled-components";
+import { getThemeProp } from "../../../../utils";
+
+const TimeWrapper = styled.div`
+	margin: auto;
+`;
+
+const TimeZoneWrapper = styled.label`
+	margin-left: 5px;
+	font-family: ${getThemeProp(["fonts", "base"], "sans-serif")};
+	font-size: 12px;
+`;
+
+const TimePickerWrapper = styled.span`
+	display: flex;
+	flex-wrap: wrap;
+	width: 100%;
+`;
+
+const TimePickerSegmentWrapper = styled.select`
+	font-family: ${getThemeProp(["fonts", "base"], "sans-serif")};
+	font-size: 12px;
+	color: #333333;
+	margin-left: 0.5em;
+	margin-right: 3px;
+`;
+
+const hourOptions = [];
+for (var i = 1; i <= 12; i++) {
+	hourOptions.push({ value: i, label: i });
+}
+
+const minOptions = [
+	{ value: "00", label: "00" },
+	{ value: "15", label: "15" },
+	{ value: "30", label: "30" },
+	{ value: "45", label: "45" },
+];
+
+const ampmOptions = [
+	{ value: "AM", label: "AM" },
+	{ value: "PM", label: "PM" },
+];
+
+const parseTime = timeStr => {
+	var time = timeStr.match(/(\d+)(?::(\d\d))?\s*(p?)/i);
+	if (!time) {
+		return new Date();
```

Comment:
Improve your  test coverage please, here is the report for your files :

  TimePicker.js                                           |   91.18 |    81.25 |     100 |   90.48 | 49,88,90,116,118,120 
  index.js                                                |     100 |    92.86 |     100 |     100 | 46

Reply:
How do I prepare a coverage report?  I'd like to verify my coverage is 100%

Reply:
@TONYFECTEAU 

Once you run npm run coverage command it generates a perfect report which located in the root folder
![image](https://user-images.githubusercontent.com/14315420/88931794-f7e9cf00-d285-11ea-8e44-a335650030ca.png)

So, what you need is just to open index.html file inside coverage folder.

Reply:
Done